### PR TITLE
Initial vSphere IPI install instructions

### PR DESCRIPTION
**Pull request by simu**
This should not be considered final, but I think it's now good enough to review and merge. We can then do another pass over it once we setup the next cluster.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/vsphere/install.adoc [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+export VSPHERE_PASSWORD=<password>
+----
+
+.vSphere resources
+[source,bash]
+----
+export VSPHERE_CLUSTER=<cluster name>
+export VSPHERE_DATACENTER=<datacenter name>
+export VSPHERE_DATASTORE=<datastore name>
+export VSPHERE_FOLDER=<folder name>
```
Comment (haasad) [inline docs/modules/ROOT/pages/how-tos/vsphere/install.adoc:67]: Is this required? Don't see it referenced anywhere else?

File: docs/modules/ROOT/pages/how-tos/vsphere/install.adoc [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+<2> This command expects that your SSH config is setup so that `ssh ${JUMPHOST_FQDN}` works without further configuration
+<3> The `openshift-install` tool respects the `https_proxy` environment variable
+====
+
+. Trust the vSphere CA certificate
++
+.Ubuntu 22.04
+[source,bash]
+----
+curl -kLo vsphere-ca.zip ${VCENTER_API_URL}/certs/download.zip <1>
```
Comment (haasad) [inline docs/modules/ROOT/pages/how-tos/vsphere/install.adoc:156]: Footnote is missing here afaics

**Review by haasad — APPROVED** [review]
LGTM in the current form :+1: we'll have to refine it once we set up the next cluster following these instructions, but looks already very complete

File: docs/modules/ROOT/pages/how-tos/vsphere/install.adoc (hunk 1 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-tos/vsphere/install.adoc:67]: It's not required by default. I've removed it for now.

File: docs/modules/ROOT/pages/how-tos/vsphere/install.adoc (hunk 2 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-tos/vsphere/install.adoc]: I figured out how to tell curl to resolve DNS names over a SOCKS5 proxy, so I've updated this section slightly.

**Review by corvus-ch — APPROVED** [review]
While it is not necessarily easy to maintain, I really like how the includes helps to understand what the common steps are and where the different platforms differ.

**Pull request merged**